gui-wm/hyprland/hyprland-9999.ebuild

```diff
@@ -20,7 +20,7 @@ fi
 
 LICENSE="BSD"
 SLOT="0"
-IUSE="X legacy-renderer +qtutils systemd"
+IUSE="hyprpm X legacy-renderer +qtutils systemd"
```

Comment:
It's probably best to have it disabled by default. That code is a mess, and really should have been a separate package (upstream).

Reply:
hyprpm has been enabled by default on Gentoo (no USE flag) since it was first introduced

It is indeed not a separate package and upstream's documentation don't mention explicit installation for hyprpm.

It does not pull many deps and I'm sure most people already have those deps present on their system (ninja, meson, cmake, git, pkgconfig).

So introducing this USE flag is not sparing the user installation of needless deps, it is merely preventing the build and installation a component of the application. That component gets built rather quickly and the binary size is low (235K + shell autocompletions)

For those reasons I think it should be enabled by default.

As a side note, this USE flag will need to be added to the metadata.xml file.

Reply:
> hyprpm has been enabled by default on Gentoo (no USE flag) since it was first introduced

So adding this USE flag disabled by default means current users of it may inadvertently end up without hyprpm on their systems, if they go through their world update too fast.

Reply:
We'd definitely want to notify users to introduce a USE flag for hyprpm. I'll leave this in draft until you and I can agree on how to break hyprpm into a USE flag or separate package and then we can ewarn/elog in pkg_postinst.

I think it's unlikely that most users would be impacted, and those who wish to install hyprpm could add the USE flag. This prevents buggy code from getting installed by default.